tests/distributions/test_distribution.py

```diff
@@ -473,25 +473,22 @@ def test_custom_dist_default_moment(self, dist_params, size, expected, dist_fn):
         assert_moment_is_expected(model, expected)
 
     def test_custom_dist_default_moment_inner_graph(self):
-        def scan_step(xtm1):
-            x = xtm1 * 2
+        def scan_step(mu):
+            x = pm.Normal.dist(mu, 1)
             x_update = collect_default_updates([x])
             return x, x_update
 
         def dist(size):
-            x0 = pm.Normal.dist(1, 1)
-
             xs, updates = scan(
                 fn=scan_step,
-                outputs_info=[x0],
-                n_steps=2,
+                sequences=[pt.ones(2)],
                 name="xs",
             )
-            return xs[-1]
+            return pt.sum(xs)
```

Comment:
`pt.sum` gives a variable for which PyMC doesn't know the logp. You can return the whole `xs` that should be fine. Would be good to use something like a `Uniform(-2, -1)` inside the Scan, for which the old `CustomDist` moment of zeros would fail. 

And then you can remove the `check_finite_logp=False`